Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -282,6 +282,22 @@ def run_checks(
           version_info[:latest] if version_info.present?
         end
 
+        # Check current and latest resources (if "--resources" flag is given)
+        # Only check current and latest versions if we have resources to check against
+        check_for_resources = check_resources && formula_or_cask.is_a?(Formula) && formula_or_cask.resources.present?
+        if check_for_resources
+          resource_version_info = formula_or_cask.resources.map do |resource|
+            resource_info = resource_version(
+              resource,
+              json:      json,
+              full_name: use_full_name,
+              verbose:   verbose,
+              debug:     debug,
+            )
+            resource_info
+          end
+        end
+
         if latest.blank?
```

Comment:
It seems like the code to handle a blank latest version for resources is not present. This situation arises when a latest version could not be found and must be implemented just like for formulae. You might have to modify `resource_version` and maybe even `status_hash` to accommodate this behaviour.

You should also keep in mind that a formula's livecheck may error out with the "unable to get versions" message, but the resource results should still be presented if possible. This would mean you'll have to rearrange/modify the code that handles a formula's blank latest version and ensure that the resource information is also output, in addition to the `status_hash` result.